osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -478,11 +478,35 @@ private void postMessage(TextBox textbox, bool newText)
                 return;
             }
 
+            bool isAction = false;
+
             if (postText[0] == '/')
             {
-                // TODO: handle commands
-                target.AddNewMessages(new ErrorMessage("Chat commands are not supported yet!"));
-                return;
+                postText = postText.Substring(1);
+                string commandKeyword = postText.Split(' ')[0];
+
+                switch (commandKeyword)
+                {
+                    case "me":
+
+                        if (!postText.StartsWith("me ") || string.IsNullOrWhiteSpace(postText.Substring(3)))

```

Comment:
not so fond of this condition. why not just (store and) check `postText.Split(' ')`'s `Length` and then do the second check on  `postText.Split(' ').Skip(1).Join()` or similar? That way we're not doing magical checks on the number of characters the command is (just in case the command itself changes, or more alternative commands with the same action are added, for instance).

Reply:
I now implemented a more generic approach for easy reusable for future commands.